### ISSUE TITLE
Intermittent Failure? Tufts::WorkflowSetup grants the workflow approval role to all members of the hyrax admin group

### ISSUE THREAD

**Issue by mark-dce** [curationexperts/epigaea#260]
commit: 1bc54457a15c7f89e57893c47fb86483034ecc6d
seed: ??? (shoot)

```
  1) Tufts::WorkflowSetup grants the workflow approval role to all members of the hyrax admin group
     Failure/Error:
       expect(role_names).to contain_exactly(
         "publishing"
       )
     
       expected collection contained:  ["publishing"]
       actual collection contained:    ["managing", "publishing"]
       the extra elements were:        ["managing"]
     # ./spec/lib/tufts/workflow_setup_spec.rb:31:in `block (2 levels) in <top (required)>'
```

**Comment by no-reply** [comment]
It's possible this one was fixed by #269.

**Comment by no-reply** [comment]
Nevermind. I have a seed for this one:
47596 on 2e3ed9e60058

**Comment by bess** [comment]
I fixed this by changing what the test is testing for. In order for our publication workflow to work, we need the "publishing" role to exist. Sometimes a "managing" role is also being created for some reason (this is the result of some confusion between application roles and workflow roles, I believe). We aren't using the "managing" role, so we don't really care whether it exists or not. I changed the test to more specifically only test whether the publishing role exists.
